src/app/templates/app/license_diff.html

```diff
@@ -53,84 +54,17 @@
-		                $("#modal-body").html(message);
-                    $("#myModal").modal({
-                      backdrop: 'static',
-                      keyboard: true,
-                      show: true
-                    });
-                    $(document).on('click','button#ok', function(event){
-                      $("#myModal").modal("hide");
-                    });
-                    $(document).off().on('click','button#showDiff', function(event){
-                      generate_text_diff(originalLicenseText.split('\n\n'), inputLicenseText.split('\n\n'));
-                    });
-				          }
-                  else if (matchType == "Perfect match" || matchType == "Standard License match") {
-                    var message = `Perfect match found! The license matches with the license ID(s): <strong>${matchIds}</strong>`
-                    $("#modal-header").addClass("green-modal");
-                    $("#modal-title").html("Found!");
-                    $("#modal-body").html(message);
-                  }
-                  else {
-                    $("#modal-header").addClass("red-modal");
-                    $("#modal-title").html("Not Found!");
-                    var message = "The given license was not found in the SPDX database.";
-                    $("#modal-body").html("<h3>" + message + "</h3>");
-                  }
-                  $("#myModal").modal({
-                        backdrop: 'static',
-                        keyboard: true, 
-                        show: true
-                    });
-                  $("#licensediffbutton").text("License diff");
-                  $("#licensediffbutton").prop('disabled', false);
-              },
-              error: function (e) {
-                  console.log("ERROR : ", e);
-                  $("#modal-header").removeClass("green-modal");
-                  try {
-                    var obj = JSON.parse(e.responseText);
-                    $("#modal-header").addClass("red-modal");
-                    $("#modal-title").html("Error!");
-                    $("#modal-body").text(obj.data);
-                  }
-                  catch (e){
-                    $("#modal-header").addClass("red-modal");
-                    $("#modal-title").html("Error!");
-                    $("#modal-body").text("The application could not be connected. Please try later.");
-                  }
-                  $("#myModal").modal({
-                          backdrop: 'static',
-                          keyboard: true, 
-                          show: true
-                  });
-                  $("#licensediffbutton").text("Check License");
-                  $("#licensediffbutton").prop('disabled', false);
-              }
-          });
+      var dict1 = {
```

Comment:
nit: better to rename `request`

Reply:
Hi @rtgdk I have made these changes let me know if anything else needs to be done in this